### DISCUSSION TITLE
TagsInput suggestions appears drastically different on firefox due to datalist

### DISCUSSION THREAD

**Discussion by blite** [filamentphp/filament#2067] [Ideas]
### Package

filament/forms

### Package Version

head

### Laravel Version

v9.6.0

### Livewire Version

v2.10.4

### PHP Version

8.1 sail docker

### Bug description

Firefox:

![image](https://user-images.githubusercontent.com/442226/161100393-1cbc2e3b-b12d-45ec-b584-7931c92f0b92.png)

Chrome:

![image](https://user-images.githubusercontent.com/442226/161100514-0c96ba2f-5ca4-4bb0-a171-72e501892465.png)

I started looking at this to fix this icon being different from the one on select boxes:

![image](https://user-images.githubusercontent.com/442226/161101068-d8f95234-aea9-47d0-9523-3fc5471a6d53.png)

that icon is caused by the <datalist> which is used for suggestions.  Apparently datalist does have some support issues https://caniuse.com/datalist . 

I think the correct course of action in the long run will be to change to using a div similar to the ones used on Select components.  This will remove the different icon as well.

**Comment by danharrin** [comment]
Hey, I'd like to avoid replacing the datalist with a JavaScript-based option as it would result in lots of duplicate code, when the browser support isn't *that* bad. Also, I don't think that the Firefox version looks that bad?
